test/integration/functional_test.go

```diff
@@ -670,7 +670,7 @@ func validateServiceCmd(ctx context.Context, t *testing.T, profile string) {
 		t.Fatalf("failed to get service url. args %q : %v", rr.Command(), err)
 	}
 	if rr.Stderr.String() != "" {
-		t.Errorf("expected stderr to be empty but got *%q*", rr.Stderr)
+		t.Logf("expected stderr to be empty but got *%q*", rr.Stderr)
```

Comment:
Add a comment and say this might be okay.
Any chance that we look for the error and if it contains the cgroup error then we only log it
Otherwise error ?

Reply:
oops I merged too quickly, do you mind adding a follow up PR to add comment why that error is okay